HelloWorld/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<manifest package="net.boondockradio.helloworld"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name" android:supportsRtl="true"
```

Comment:
@eoinfogarty I agree.

@nbhd 
If you support `RTL`, write `android:supportsRtl="true"`.
If not , delete it. default value is `false`.
see https://developer.android.com/guide/topics/manifest/application-element.html